ci/ci.py

```diff
@@ -368,7 +369,20 @@ def get_branch():
         branch = call("git rev-parse HEAD")
     return branch
 
+def is_workflow_enabled_in_repo():
+    if not is_repo_private():
+        return True # all workflows are enabled by default
+
+    workflow = os.environ.get('GITHUB_WORKFLOW', '')
+    if workflow in ('CI - Main',):
```

Comment:
Should the comma be there?

Reply:
If it's not, then it won't become an immutable list, but just a value with parenthesis around it

> value = (value)

vs

> immutable_list = (value,)

Reply:
Ah, ofc, how silly of me :-)